examples/pi_series_omega_control.py

```diff
@@ -4,42 +4,70 @@
 """
 from agents.simple_controllers import Controller
 import time
-import sys, os
+import sys
+import os
 sys.path.append(os.path.abspath(os.path.join('..')))
 import gym_electric_motor as gem
 from gym_electric_motor import reference_generators as rg
 from gym_electric_motor.visualization import MotorDashboard
 
+# This example shows the behavior of GEM when using standard linear controllers
+# In the following, we use a PI controller to control the speed of a series DC motor
 
 if __name__ == '__main__':
+
+    # Define the drive environment
     env = gem.make(
-        'DcSeriesCont-v1',   # replace with 'DcSeriesDisc-v1' for discrete controllers
-        # Pass an instance
-        #visualization=MotorDashboard(plotted_variables='all', visu_period=1),
-        visualization=MotorDashboard(plots=['i', 'reward', 'action_0', 'mean_reward'], dark_mode=False),
+        # Define the series DC motor with continuous-control-set
+        'DcSeriesCont-v1',
+
+        # Set the electric parameters of the motor
         motor_parameter=dict(r_a=15e-3, r_e=15e-3, l_a=1e-3, l_e=1e-3),
-        # Take standard class and pass parameters (Load)
+
+        # Set the parameters of the mechanical polynomial load (the default load class)
         load_parameter=dict(a=0.01, b=.1, c=0.1, j_load=.06),
 
-        # Pass a string (with extra parameters)
-        ode_solver='scipy.solve_ivp', solver_kwargs=dict(),
-        # Pass a Class with extra parameters
+        # Define a reference generator
+        # In this example, we permit random switching between sinusoidal, wiener and step reference with
+        # probabilities 0.1, 0.8 and 0.1 respectively.
+        # super_episode_length defines minimum and maximum time steps between switching references
         reference_generator=rg.SwitchedReferenceGenerator(
             sub_generators=[
                 rg.SinusoidalReferenceGenerator, rg.WienerProcessReferenceGenerator(), rg.StepReferenceGenerator()
             ], p=[0.1, 0.8, 0.1], super_episode_length=(1000, 10000)
-        )
```

Comment:
@atra94 
I think without knowing GEM, one is unable to see from the code that this really is supposed to be a speed control scenario. Can we make that more obvious?